### PR TITLE
Add missing tube number in DTO marshalling

### PR DESCRIPTION
**Pull request by apmasell**

### CODE REVIEW

File: miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/Dtos.java [hunk 1]
```diff
@@ -237,6 +238,7 @@ public static SampleAdditionalInfo to(SampleAdditionalInfoDto from) {
     to.setTubeNumber(from.getTubeNumber());
     to.setVolume(from.getVolume());
     to.setConcentration(from.getConcentration());
+    to.setTubeNumber(from.getTubeNumber());
     return to;
```
Comment (callunity) [inline miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/Dtos.java]: It is already set in this method, three lines up.

Comment (apmasell) [inline miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/Dtos.java]: Fixed.

**Comment by callunity** [comment]
:+1:

**Comment by djcooke** [comment]
:+1:

**Pull request merged**